README.md

```diff
@@ -35,18 +34,18 @@ This agent can be used for the VSTS service and it replaces the node agent for T
 ## Get Started
 
 ### First, add the build account to the proper roles
-    
+
```

Comment:
why are you taking off trailing spaces?  In markdown 2 spaces at end of line is a CR